lib/buildFactory.js

```diff
@@ -306,6 +307,51 @@ class BuildFactory extends BaseFactory {
         return super.list(config);
     }
 
+    /**
+     * Get build statuses for jobs
+     * @method getBuildStatuses
+     * @param  {Object}   config                  Config object
+     * @param  {Array}    config.jobIds           Jobs to get build statuses for
+     * @param  {Number}   config.offset           Number of build statuses to skip
+     * @param  {Number}   config.numBuilds        Number of build statuses to return per job
+     * @return {Promise}
+     */
+    getBuildStatuses(config) {
+        config.offset = config.offset || 0;
```

Comment:
probably shouldn't override function params like this
might be better to create a new var like this:
```suggestion
        const offset = config.offset || 0;
```

Reply:
Done, thanks!